src/Features/Core/Portable/StackTraceExplorer/IStackFrameParser.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Diagnostics.CodeAnalysis;
+
+namespace Microsoft.CodeAnalysis.StackTraceExplorer
+{
+    internal interface IStackFrameParser
+    {
+        bool TryParseLine(string line, [NotNullWhen(returnValue: true)] out ParsedFrame? parsedFrame);
```

Comment:
fwiw, i think `NotNullWhen(true)` is idiomatic.  so i would drop the `returnValue`.  but it's up to you.

Reply:
🤷‍♂️ it's habit for me after seeing it enough. I'm not too opinionated, but I'm going to leave it just because it looks more natural to me.